subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/maven/PomProjectInitDescriptor.java

```diff
@@ -86,29 +74,12 @@ public String getSourceBuildDescription() {
         return "Maven";
     }
 
-    @Override
-    public void configureClasspath(ProjectInternal.DetachedResolver detachedResolver, ObjectFactory objects) {
-        DependencyHandler dependencies = detachedResolver.getDependencies();
-        Configuration config = detachedResolver.getConfigurations().detachedConfiguration(
-            dependencies.create("org.apache.maven:maven-core:" + MAVEN_VERSION),
-            dependencies.create("org.apache.maven:maven-plugin-api:" + MAVEN_VERSION),
-            dependencies.create("org.apache.maven:maven-compat:" + MAVEN_VERSION),
-            dependencies.create("org.apache.maven.wagon:wagon-http:" + MAVEN_WAGON_VERSION),
-            dependencies.create("org.apache.maven.wagon:wagon-provider-api:" + MAVEN_WAGON_VERSION),
-            dependencies.create("org.eclipse.aether:aether-connector-basic:" + AETHER_VERSION),
-            dependencies.create("org.eclipse.aether:aether-transport-wagon:" + AETHER_VERSION)
-        );
-        config.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage.class, Usage.JAVA_RUNTIME));
-        detachedResolver.getRepositories().mavenCentral();
-        mavenClasspath = config;
-    }
-
     @Override
     public void generate(InitSettings initSettings) {
         IncubationLogger.incubatingFeatureUsed("Maven to Gradle conversion");
         try {
             Settings settings = settingsProvider.buildSettings();
-            executor.classLoaderIsolation(config -> config.getClasspath().from(mavenClasspath))
+            executor.noIsolation()
```

Comment:
I wonder if we'd want to keep the maven dependencies isolated. It would mean including the jars in the distro but not including them in the Gradle _core and plugins_ classpath and resolving them from the distro using the module registry / classpath provider mechanism instead.

WDYT @big-guy, @ljacomet ?